### PR TITLE
Update ban message

### PR DESCRIPTION
**Pull request by captainmeta4**
As per this modtalk discussion: https://www.reddit.com/r/modtalk/comments/3g6786/-/ctv96dt

Per the contribution guidelines, I (and other mods, if you read that thread) feel this change is meaningful enough to warrant a string change.

### CODE REVIEW

File: r2/r2/lib/system_messages.py [hunk 1]
```diff
@@ -125,10 +125,10 @@ def send_ban_message(subreddit, mod, user, note=None, days=None):
         message += "\n\n" + blockquote_text(note)
 
     message += "\n\n" + ("you can contact the moderators regarding your ban "
-        "by replying to this message. **warning**: using other accounts to "
-        "circumvent a subreddit ban is considered a violation of reddit's "
-        "[site rules](/rules) and can result in being banned from reddit "
-        "entirely.")
+        "by replying to this message. \n\n"
+        "**warning**: if you use another account to circumvent this subreddit ban, "
+        "it would be considered a violation of reddit's [site rules](http://reddit.com/rules) "
```
Comment (xiongchiamiov) [inline r2/r2/lib/system_messages.py]: I don't have any comment on this PR in general, but this link should probably stay `/rules`: that will keep them on the same subdomain and protocol.

Comment (captainmeta4) [inline r2/r2/lib/system_messages.py]: Reverted to `/rules`

**Comment by bsimpson63** [comment]
Can you explain how this wording will be interpreted differently? I don't really understand.

**Comment by nty-** [comment]
It looks like the new wording is supposed to emphasize the hypothetical nature of the warning.

I personally don't like how the change, however, since I think the passive voice weakens the overall message.

**Comment by bsimpson63** [comment]
OK, then I'm going to close this pull request. Sorry.

**Pull request closed without merging**